Add issue templates

cmark opened the pull request.

cmark committed:
Add issue templates

apeteri commented on .github/ISSUE_TEMPLATE/bug_report.md:
```diff
@@ -0,0 +1,27 @@
+
+Useful things to include in the report:
+* Example Request/Response pairs
+* Example `snowowl.yml` configuration
```
I don't have exact words for it, but we could mention two things:

* The submitter should make sure not to expose any credentials accidentally when including files or snippets
* The reproduction steps, requests, and responses should form a _minimal_ reproducible example, if possible - it is harder to try and spot an error in megabytes of logs than looking at a smallish configuration object.

cmark replied:
Yeah good ideas, will include them soon. 👍

cmark committed:
[releng] add bug_report template suggestions

NOTE about exposing credentials.
Some additional bullet points to the useful things to include section.

apeteri reviewed (approved):
:+1:
